docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -39,6 +39,79 @@ The Fairlearn package is most applicable to two kinds of harms:
   information are extended or withheld. Examples include varying accuracy in
   face recognition, document search, or product recommendation.
 
+Concept Glossary
+----------------------------
+
+The concepts outlined in this glossary are relevant to sociotechnical contexts. 
+
+Construct Validity
+^^^^^^^^^^^^^^^^^^
+
+We explore the concept of construct validity, as described in in Measurement and Fairness [#4]_ , and as it relates to sociotechnical contexts. This in intended to illuminate the concept to readers who are working on solutions in sociotechnical contexts and may also be referenced in explorations that detail how to apply the concept to specific scenarios.
+
+Key Terms 
+~~~~~~~~~
+
+- **Sociotechnical context** – A context in which both societal and technical aspects shape who might be harmed by AI systems and how (see fairlearn.ai for more detailed definition).
+
+- **Unobservable theoretical construct** – An idea or concept that is unobservable and cannot be directly measured but must instead be inferred through observable measurements defined in a measurement model. 
```

Comment:
I am not sure whether people unfamiliar with the idea of theoretical constructs would understand this definition. I am also not a big fan of referring to concepts you have not introduced yet. Maybe something like: "An abstract idea or concept that is unobservable and cannot be measured directly." I think it would also help if we add the key term examples **directly** under the associated definition.

Reply:
Given the current structure of the examples (where showcase both unobservable theoretical constructs and an measurement models within the same example), how might you suggest restructuring this show that the key term examples appear **directly** under the associated definition. This was my initial structure, but I found that the key term examples were easier to follow if both unobserved tc and measurement model were combined into one example. 

If you think having it directly under would be clearer, could you illustrate how we'd do this

Reply:
I'd also be in favor of at least introducing the examples after each one: something like "(e.g, teacher effectiveness, socioeconomic status, or fairness)". You can always call back to those later with the specific measurements, but I think that would help to ground it a bit!

Reply:
Also agreed that we may want to drop the "defined in a measurement model" since that gets introduced later